internal/service/eventtrigger/data_source_event_trigger_test.go

```diff
@@ -39,12 +39,12 @@ func TestAccEventTriggerDS_basic(t *testing.T) {
 	resource.Test(t, resource.TestCase{
 		PreCheck:                 func() { acc.PreCheck(t) },
 		ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
-		CheckDestroy:             testAccCheckMongoDBAtlasEventTriggerDestroy,
+		CheckDestroy:             checkDestroy,
```

Comment:
please use the new random name functions,   in general all ocurrences for "acctest." can be changed, e.g:

Name:       acc.RandonName(),